lib/directory-view.coffee

```diff
@@ -9,6 +9,8 @@ class DirectoryView extends HTMLElement
     @subscriptions.add @directory.onDidDestroy => @subscriptions.dispose()
     @subscribeToDirectory()
 
+    @draggable = true
```

Comment:
Ah, of course. For a second I was thinking this was a SpacePen view but it's a custom element. Makes sense.